### PR TITLE
fixed timezone thingies

### PR DESCRIPTION
**Pull request by jiisloth**

### CODE REVIEW

File: vj_cms/cms_plugins.py [hunk 1]
```diff
@@ -60,25 +60,29 @@ def render(self, context, instance, placeholder):
         day_was = ""
         day = []
         for game in games:
+            duration = (game.end_time - game.start_time).total_seconds()
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: This is already stored in the GameInfo model as `estimate`

File: vj_cms/cms_plugins.py [hunk 2]
```diff
@@ -60,25 +60,29 @@ def render(self, context, instance, placeholder):
...
+            duration = str(int(duration // 3600)) +"h " + str(int((duration % 3600) // 60)) + "min"
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: This is implemented as a template filter named `duration` in `vj_cms/templatetags/vj_extras.py`.
Usage:
```
{% load vj_extras %}
{{ game.estimate|duration }}
```

Rather use that filter instead, modify if needed

File: vj_cms/cms_plugins.py [hunk 3]
```diff
@@ -60,25 +60,29 @@ def render(self, context, instance, placeholder):
...
             data = {
                 "game": game.game,
                 "img": game.img_filename,
                 "player": game.player,
                 "player_twitch": game.player_twitch,
-                "category": game.category
+                "category": game.category,
+                "start_time":  game.start_time,
+                "end_time":  game.end_time,
+                "duration":  duration
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: This context object is basically a copy of the game model object, consider passing the whole object in

File: vj_cms/cms_plugins.py (hunk 2 above)
Comment (jiisloth) [reply vj_cms/cms_plugins.py]: Yes. Did not notice it at all. norppaLikeThis

File: vj_cms/cms_plugins.py (hunk 3 above)
Comment (jiisloth) [reply vj_cms/cms_plugins.py]: Considered but did not implement

**Pull request merged**